src/Akka.Persistence.Sql.Linq2Db/Config/SnapshotTableColumnNames.cs

```diff
@@ -46,31 +48,5 @@ public override int GetHashCode()
             return HashCode.Combine(PersistenceId, SequenceNumber, Created,
                 Snapshot, Manifest, SerializerId);
         }
-
-        private static readonly string FallBack =
```

Comment:
Baking hardwired HOCON config is a bad idea, this creates 4 copies of the same HOCON config in 4 different place